pkg/meta/annotations.go

```diff
@@ -140,6 +140,9 @@ const (
 	// annotation is not provided the default value "dbadmin" will be used.
 	SuperuserNameAnnotation   = "vertica.com/superuser-name"
 	SuperuserNameDefaultValue = "dbadmin"
+
+	// Annotation to control the termination grace period for vertica pods.
+	TerminationGracePeriodSecondsAnnotaton = "vertica.com/termination-grace-period-seconds"
```

Comment:
Is there any use for this annotation other than reproducing that issue?

Reply:
Yes, only for test purposes. There are no plans to externalize this annotation.